maps/tests/script.js

```diff
@@ -1,40 +1,42 @@
+///<reference path="../../front/src/iframe_api.ts" />
```

Comment:
@moufmouf  this allows type checking in javascript files if checkJs is enabled in tsConfig